docs/Documentation/Drivers/Python/PythonDriver.md

```diff
@@ -0,0 +1,55 @@
+VIC Python Driver
+================
+
+The VIC Python driver exposes the `vic_run` and `vic_driver_shared` function APIs through a Python interface. It may be used to test individual algorithms or to wrap the `vic_run` function.
```

Comment:
exposes is what I mean though.  It just gives the user access to the C functions.